tests/src/test/scala/runtime/actionContainers/PythonActionContainerTests.scala

```diff
@@ -208,127 +210,6 @@ class PythonActionContainerTests extends BasicActionRunnerTests with WskActorSys
       })
   }
 
-  it should "run zipped Python action containing a virtual environment" in {
```

Comment:
moved to action loop runtime. This class is going to get refactored in a subsequent PR.